### PR TITLE
Add driver and daemon support for Razer BlackWidow X Ultimate

### PR DESCRIPTION
**Pull request by dfrankland**
http://www.razerzone.com/gaming-keyboards-keypads/razer-blackwidow-x-ultimate

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ Website: [Here](https://terrycain.github.io/razer-drivers/)
 | Razer Blade Pro (Late 2016)                   |  1532:0210  |   ✔    |   ✔    |
 | Razer BlackWidow Ultimate 2016                |  1532:0214  |   ✔    |   ✔    |
 | Razer BlackWidow X Chroma                     |  1532:0216  |   ✔    |   ✔    |
+| Razer BlackWidow X Ultimate                   |  1532:0216  |   ✔    |   ✔    |
```
Comment (terricain) [inline README.md]: Should be 0217

**Comment by terricain** [comment]
What options / effects can the keyboard do? your still missing alot.

**Review by terricain — CHANGES_REQUESTED** [review]

**Review by terricain — APPROVED** [review]

**Comment by dfrankland** [comment]
There aren't very many options on this keyboard. All it has are the per-key lighting options, recordable macros, and gaming mode.

As far as lights go there are really only these options, in green color, that are built in:
*   Brightness
*   Static On / Off
*   Starlight
*   Ripple
*   Breathe
*   Reactive
*   Wave

**Comment by terricain** [comment]
It can light up individual keys but technically they're only green right?

**Comment by dfrankland** [comment]
Exactly.

**Comment by terricain** [comment]
Ok open an issue as we'll need to do some driver testing before we merge stuff.

**Pull request closed without merging**